flow/metrics.yaml

```diff
@@ -8,7 +8,7 @@ components:
         enable:
```

Comment:
Before finalizing the OTG API model, can also finalize the gNMI model as well, to avoid any conflict between these two models and the possibility of rework in the future?